src/test/java/ezschedule/logic/commands/RecurCommandTest.java

```diff
@@ -0,0 +1,292 @@
+
+        // EAT has clashing time with ART
+        Event eat = new EventBuilder().withName("Eating contest")
+                .withDate("2023-10-01").withStartTime("12:00").withEndTime("13:00").build();
+        model.addEvent(eat);
+
+        assertCommandFailure(validMonthlyRecurCommandStub, model,
+                String.format(RecurCommand.MESSAGE_FAILURE_EVENT_CLASH, clashDate.getDay(), month));
+    }
+
+    @Test
+    public void execute_validIntToStringMonths_success() {
+        // valid months [1, 12]
+        String expectedMonthJan = "January";
+        String actualMonthJan = validDailyRecurCommandStub.intToStringMonth(1);
+
+        String expectedMonthFeb = "February";
+        String actualMonthFeb = validDailyRecurCommandStub.intToStringMonth(2);
+
+        String expectedMonthMar = "March";
+        String actualMonthMar = validDailyRecurCommandStub.intToStringMonth(3);
+
+        String expectedMonthApr = "April";
+        String actualMonthApr = validDailyRecurCommandStub.intToStringMonth(4);
+
+        String expectedMonthJun = "June";
+        String actualMonthJun = validDailyRecurCommandStub.intToStringMonth(6);
+
+        String expectedMonthDec = "December";
+        String actualMonthDec = validDailyRecurCommandStub.intToStringMonth(12);
+
+        assertEquals(expectedMonthJan, actualMonthJan);
+        assertEquals(expectedMonthFeb, actualMonthFeb);
+        assertEquals(expectedMonthMar, actualMonthMar);
+        assertEquals(expectedMonthApr, actualMonthApr);
+        assertEquals(expectedMonthJun, actualMonthJun);
+        assertEquals(expectedMonthDec, actualMonthDec);
+    }
+
+    @Test
+    public void execute_invalidIntToStringMonths_success() {
+        String invalidMessage = "No such month";
+
+        // invalid months [..., 0]
+        String actualMonthZero = validDailyRecurCommandStub.intToStringMonth(0);
+        String actualMonthNegative = validDailyRecurCommandStub.intToStringMonth(-10);
+
+        // invalid months [13, ...]
+        String actualMonthThirteen = validDailyRecurCommandStub.intToStringMonth(13);
+        String actualMonthHundred = validDailyRecurCommandStub.intToStringMonth(100);
+
+
+        assertEquals(invalidMessage, actualMonthZero);
+        assertEquals(invalidMessage, actualMonthNegative);
+        assertEquals(invalidMessage, actualMonthThirteen);
+        assertEquals(invalidMessage, actualMonthHundred);
+    }
+
+    @Test
+    public void equals_success() {
+        final RecurCommand standardCommand =
+                new RecurCommand(INDEX_FIRST_EVENT, new Date(VALID_DATE_A),
+                        new RecurFactor(VALID_RECUR_FACTOR_DAY));
+
+        // same values -> returns true
+        RecurCommand commandWithSameValues =
+                new RecurCommand(INDEX_FIRST_EVENT, standardCommand.getEndDate(),
+                        standardCommand.getFactor());
+        assertEquals(standardCommand, commandWithSameValues);
+
```

Comment:
I think you followed my changes, which is wrong.
1. change equals_success() to equals()
2. instead of assertEquals(...) use assertTrue(standardCommand.equals(commandWithSameValues))

Using assertEquals will actually make use of the equals method unlike assertEquals, which also contributes to the code coverage

Reply:
you mean unlike assertTrue?

Reply:
updated

Reply:
> Using assertEquals will actually make use of the equals method unlike assertEquals, which also contributes to the code coverage

I think we only need to use this once somewhere. Apart from that, I feel that it would be better to use `assertEquals` as it make more sense to whoever reading the code. (Doesn't pause the reader and make them question why you are using a roundabout way when there is a cleaner `Assertion`  method to use).